flexmeasures/data/models/generic_assets.py

```diff
@@ -355,6 +359,8 @@ def search_annotations(
         annotation_type: str = None,
         include_account_annotations: bool = False,
         as_frame: bool = False,
+        sensor_id: int = None,
```

Comment:
Shouldn't this be named `asset_id` (or `asset_or_sensor_id`)?

Reply:
Actually, I don't think we need it. See my comment about using calling this method on the instance rather than the class.